manifest.json

```diff
@@ -6,20 +6,20 @@
     "en": "News feed (RSS/Atom) reader and aggregator.",
     "fr": "Lecteur de flux d’actualité utilisant les protocoles RSS et Atom."
   },
-  "version": "20200618~ynh1",
+  "version": "20200906~ynh1",
   "url": "http://tt-rss.org",
   "license": "GPL-3.0-only",
   "maintainer": {
     "name": "YunoHost Contributors",
     "email": "apps@yunohost.org"
   },
   "requirements": {
-    "yunohost": ">= 3.5"
+    "yunohost": ">= 3.8.1"
   },
   "multi_instance": true,
   "services": [
       "nginx",
-      "php5-fpm",
+      "php7.0-fpm",
```

Comment:
Hello ! Shouldn't the service be named "php7.3-fpm" ?

Reply:
I once tried to set it to php7.3-fpm and the installscript wasn't so happy about it. I think is it a declaration of services with no real use in terms of version specification.